src/OsiCbcSolverInterface.cpp

```diff
@@ -1,15 +1,16 @@
 #include "CbcConfig.h"
+#include "OsiCbcSolverInterface.hpp"
 
-// CBC_VERSION_MAJOR defined for Cbc > 2.5
-#ifndef CBC_VERSION_MAJOR
-#include "OsiCbcSolverInterface_2_5.cpp"
-#elif CBC_VERSION_MAJOR == 2 && CBC_VERSION_MINOR <= 8
-#include "OsiCbcSolverInterface_2_8.cpp"
-#elif CBC_VERSION_MAJOR == 2 && CBC_VERSION_MINOR == 9
-#include "OsiCbcSolverInterface_2_9.cpp"
-#else
-#error "Cyclus cannot yet handle your version of CoinCBC. Please open an issue with your CoinCBC version."
-#endif
+// // CBC_VERSION_MAJOR defined for Cbc > 2.5
+// #ifndef CBC_VERSION_MAJOR
+// #include "OsiCbcSolverInterface_2_5.hpp"
+// #elif CBC_VERSION_MAJOR == 2 && CBC_VERSION_MINOR <= 8
+// #include "OsiCbcSolverInterface_2_8.hpp"
+// #elif CBC_VERSION_MAJOR == 2 && ( CBC_VERSION_MINOR == 9 || CBC_VERSION_MINOR == 10)
+// #include "OsiCbcSolverInterface_2_9_10.hpp"
+// #else
+// #error "Cyclus cannot yet handle your version of CoinCBC. Please open an issue with your CoinCBC version."
+// #endif
```

Comment:
I think we need to revert this change, right? and make sure to include the `*.cpp` files here.